index.html

```diff
@@ -26,14 +28,17 @@
           <span><a class="logo" href="#">Houssam</a></span>
         </div>
         <ul class="hide_mobile">
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#">About</a></li>
-          <li><a href="#">contact</a></li>
+          <li><a class="mnitem" href="#portfolio">Portfolio</a></li>
+          <li><a class="mnitem" href="#aboutme">About</a></li>
+          <li><a class="mnitem" href="#contact">contact</a></li>
         </ul>
         <button class="hamburger">
           <span><i class="fa fa-bars fa-2x"></i></span>
         </button>
       </nav>
+      <button class="close">
+        <strong>x</strong>
+      </button>
```

Comment:
Please increase the size of X so that it looks clear and according to the design.

Reply:
This issue is well resolved! ✔️